docs/how-to-guides/curating-the-editor-experience.md

```diff
@@ -12,7 +12,11 @@ Users have the ability to lock and unlock blocks via the editor. The locking UI
 
 ![Image of locking interface](https://raw.githubusercontent.com/WordPress/gutenberg/HEAD/docs/assets/Locking%20interface.png?raw=true)
 
-Keep in mind that each block you want to lock will need to be individually locked as desired. There is not a way to mass lock blocks currently. 
+Keep in mind that you can apply the locking options to blocks inside of the initial block you are locking by turning on the "Apply to all blocks inside" option but you cannot mass lock blocks otherwise. 
```

Comment:
This is almost of a bit of a tongue-twister with all the mentions of "lock" and "block" in a single sentence. I definitely had to read it a couple of times to make sure I understood.

Not sure if this is a lot better, but just a rough go at it:

> Keep in mind that you can apply locking options to blocks nested inside of a containing block by turning on the "Apply to all blocks inside" option. However, you cannot mass lock blocks otherwise.

Reply:
Updated! Thank you for the rewrite option.